source/examples/time-series-forecasting-with-hpo/preprocessing_part1.ipynb

```diff
@@ -0,0 +1,3201 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import pandas as pd
</pre><p>Where is pandas being used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Removed.</p>